libs/langchain/langchain/callbacks/openai_info.py

```diff
@@ -90,6 +98,8 @@ def standardize_model_name(
 
     """
     model_name = model_name.lower()
+    if ".ft-" in model_name:
+        return model_name.split(".")[0] + "-azure-finetuned"
```

Comment:
This should work, however allow me to suggest splitting on `".ft"` instead of a single dot, which might be part of the model's name (in the future).

Reply:
also for safety/clarity could we update the below check in line 103 to `if ":ft-" in model_name`

Reply:
@baskaryan @mspronesti yes I agree, Ill fix it and push 🙃